README.md

```diff
@@ -62,3 +62,6 @@ prints panics, fatal errors, and trace/debug info. All in all this can make bina
 Note: if `-tiny` is passed, no panics, fatal errors will ever be printed, but they can
 still be handled internally with `recover` as normal. In addition, the `GODEBUG` 
 environmental variable will be ignored.
+
+### Contributing
+We actively seek new contributors, if you would like to contribute to garble use the [CONTRIBUTING.md](https://github.com/burrowers/garble/blob/master/CONTRIBUTING.md) as a starting point.
```

Comment:
please follow the formatting in the rest of the file. An empty line after the section, and line wrapping.

Also, you can just link to the file via `[CONTRIBUTING.md](CONTRIBUTING.md)`. Otherwise you're forcing all branches/commits/etc to link to master at github.